src/injected-content/scss/_theatermode.scss

```
@@ -110,4 +110,12 @@ b-channel-page-wrapper .stage:not(.aspect-16-9).me-video-stage{
     position: relative !important;
 }
 
+/* Co-Stream Theatermode top-bar fix*/
+.costream-rollout {
+	top: 59px !important;
+}
+
+.costream-rollout::before {
+	background: 0 !important;
```

It might be good to have the background be a very transparent black, so the buttons and text still stand out if the stream video is very light.

```suggestion
	background: rgba(0, 0, 0, 0.25) !important;
```